Makefile

```diff
@@ -27,6 +27,11 @@ BINARIES    ?= daprd placement operator injector sentry
 HA_MODE     ?= false
 # Force in-memory log for placement
 FORCE_INMEM ?= true
+# Go's build tags:
+# all_components - includes all components in Dapr sidecar
+# stable_components - includes all stable components in Dapr sidecar
+DAPR_SIDECAR_FLAVOR ?= all
```

Comment:
Making it less verbose for assets but keep verbose in the code for clarity to devs.